### PR TITLE
Total number of results in header

### PR DESCRIPTION
**Pull request by cristianc-ty**
The header will now correctly show the total number of results

### CODE REVIEW

File: frontend/js/common.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 const IMAGES_SIZE = 20;
-const OTA_DEMO_API_URL = "https://ota-demo.integration.nbg1-c01-stag.hcloud.trustyou.net"
+const OTA_DEMO_API_URL = "http://localhost:8000"
```
Comment (harpaj) [inline frontend/js/common.js]: will that work when hosted on Github?

File: backend/ota_demo_api/repository/search_repository.py [hunk 2]
```diff
@@ -19,150 +87,166 @@ async def fetch(self, search_data: SearchRequest) -> Optional[List[ClusterSearch
...
-            WHERE cs.datapoint = 'oall' and cs.trip_type = 'all' and cs.language = 'all' and sr.rn = 1
         """
 
-        data_points = []
-        if search_data.categories or search_data.hotel_types:
-            data_points += search_data.categories or []
-            data_points += search_data.hotel_types or []
+        if is_count_query:
+            query += """
+                SELECT COUNT(*)
```
Comment (duynguyen-ty) [inline backend/ota_demo_api/repository/search_repository.py:158]: [minor] I suggest to define `select_column_list` for if/else
and then we can use `query += f"""SELECT {select_column_list} FROM search_results ...."""` to reuse all other query elements.

Comment (cristianc-ty) [reply backend/ota_demo_api/repository/search_repository.py:158]: Good idea, refacted.

File: frontend/js/common.js (hunk 1 above)
Comment (cristianc-ty) [reply frontend/js/common.js]: Good catch, forgot to switch the URL from local to integration after testing with API changes.

**Review by harpaj — APPROVED** [review]

**Review by duynguyen-ty — APPROVED** [review]

**Pull request merged**